### DISCUSSION TITLE
Ayuda en Ggit Hub

### DISCUSSION THREAD

**Discussion by FernandoMoralesH** [LaunchX-InnovaccionVirtual/CursoIntroPython#189] [General]
Hola; 

Me podrían ayudar a corregir un error o ver cual es la causa de por que no puedo visualizar un archivo en Git Hub al momento de presionar en Open Preview me sale un error: 

 Error loading webview: Error: Could not register service workers: SecurityError: The operation is insecure.

Gracias por su apoyo

![2022-02-12](https://user-images.githubusercontent.com/99518846/153717887-79a8471a-f110-4d21-be64-615de9b2ffe5.png)
.

**Comment by Victorhugohc** [comment]
El archivo que quieres abrir tiene como extensión .md (Markdown) Si quieres visualizar lo que hay en el archivo debes instalar la siguiente extensión:

Name: Office Viewer(Markdown Editor)
VS Marketplace Link: https://marketplace.visualstudio.com/items?itemName=cweijan.vscode-office

Da click en el enlace o buscalo dentro de las extensiones de VScode
